bool Gate
float32 yaw_Gate
float32 pitch_Gate
float32 distance_Gate

#### Gman ####
bool Gman
float32 yaw_Gman
float32 pitch_Gman
float32 distance_Gman

bool Badge
float32 yaw_Badge
float32 pitch_Badge
float32 distance_Badge

bool Axe
float32 yaw_Axe
float32 pitch_Axe
float32 distance_Axe

#### Bootlegger ####
bool Bootlegger
float32 yaw_Bootlegger
float32 pitch_Bootlegger
float32 distance_Bootlegger

bool Tommygun
float32 yaw_Tommygun
float32 pitch_Tommygun
float32 distance_Tommygun

bool Money
float32 yaw_Money
float32 pitch_Money
float32 distance_Money
